test/test_util.py

```diff
@@ -455,7 +457,26 @@ def test_image_decoder_b64(self):
 
     @patch("weaviate.util.uuid_lib")
     def test_generate_uuid5(self, mock_uuid):
-
         result = generate_uuid5("TestID!", "Test!")
         self.assertIsInstance(result, str)
         mock_uuid.uuid5.assert_called()
+
+    def test_parse_ver_str(self):
+        assert parse_ver_str("v1.18.1") == parse_ver_str("1.18.1")
+        assert parse_ver_str("v1.18.1") == parse_ver_str("1.18.5")
+        assert len(parse_ver_str("v1.18.1")) == 2
+        assert len(parse_ver_str("1.1.5")) == 2
+        assert len(parse_ver_str("2")) == 2
+        assert parse_ver_str("1.1.5") == (1, 1)
+        assert parse_ver_str("v1.18.3") == (1, 18)
+        assert parse_ver_str("2") == (2, 0)
+        assert parse_ver_str("2.0.0") == (2, 0)
+        assert parse_ver_str("v2.05.5") == (2, 5)
+
+    def test_is_weaviate_too_old(self):
```

Comment:
Can you use parametrization instead of a for loop?

best way to do it is to just unintend it (so it is a pytest-test) and then use:
```
@pytest.mark.parametrize("version,too_old", [("v1.0.0", True), ("v1.18.3", False)])
def test_is_weaviate_too_old(version,too_old):
    ....test....
```

Reply:
Yup, done :)